hyrax.gemspec

```diff
@@ -91,6 +91,7 @@ EOF
   spec.add_development_dependency 'shoulda-matchers', '~> 3.1'
   spec.add_development_dependency 'rails-controller-testing', '~> 0'
   spec.add_development_dependency 'webmock'
+  spec.add_development_dependency 'i18n-tasks'
```

Comment:
Do we need to require this in the app if it's a dependency?  If not, is it truly a dependency?